scripts/start/run.R

```diff
@@ -259,7 +259,7 @@ run <- function() {
   # make sure the renv used for the run is also used for generating output
   if (!is.null(renv::project())) {
     if (normalizePath(renv::project()) != normalizePath(outputdir)) {
-      warning("loaded renv=", normalizePath(renv::project()), " and outputdir=", normalizePath(outputdir), " must be equal.")
```

Comment:
I strongly oppose that. If the output generation runs with some renv from the main folder or whatever, so what. This is not ideal for reproducibility (therefore: warn), but most often not a real drama. See also https://github.com/magpiemodel/magpie/pull/688

Reply:
No problem. I thought we wanted it this way once the bug is fixed, but I'll change it back.